src/cloud-providers/libvirt/libvirt.go

```diff
@@ -27,6 +27,12 @@ const (
 	GetDomainIPsRetries = 20
 	// The sleep time between retries to get the domain IP addresses
 	GetDomainIPsSleep = time.Second * 3
+	// Default Memory size
+	LIBVIRT_DEF_MEM_SIZE = uint(8)
+	// Default CPU size
+	LIBVIRT_DEF_CPU_SIZE = uint(2)
+	// Convert Memory size from MegaBytes to GigaBytes
+	LIBVIRT_CONV_MEM = 1000
```

Comment:
The kata annotation is in MiB (https://github.com/kata-containers/kata-containers/blob/main/docs/how-to/how-to-set-sandbox-config-kata.md#hypervisor-options) and the domainXML we create is in GiB (https://github.com/confidential-containers/cloud-api-adaptor/blob/3dc40c0073d2c24235806eb8718e7b130217c3e1/src/cloud-providers/libvirt/libvirt.go#L335), so the comment and conversion constant needs correcting.